src/reducers/feeds.js

```diff
@@ -42,7 +46,11 @@ export function getIlkData(feeds, ilkKey) {
     liquidationRatio: ilkData[LIQUIDATION_RATIO],
     liquidationPenalty: ilkData[LIQUIDATION_PENALTY],
     ilkRate: ilkData[ILK_RATE],
-    stabilityFee: ilkData[RATE]
+    stabilityFee: ilkData[RATE],
+    ilkDebtAvailable: sub(
+      ilkData[DEBT_CEILING],
+      getIlkDebtAmount(ilkData[ILK_ART], ilkData[ILK_RATE])
```

Comment:
what do you think about using the [`debtValue`](https://github.com/makerdao/dai.js/blob/37e38deeaf9414d26fd7d0c8ba5cf77926b3d33a/lib/dai-plugin-mcd/src/math.js#L59) math function from the sdk here? if the units are off b/c of the normal `sub` w/ `ilkData[DEBT_CEILING]`, it might also be worth adding a `debtAvailableValue` fnc to the sdk

Reply:
cool, I'll give that a shot

Reply:
Nice, using `debtValue` produces the same value there. I'm trying to think of the best way to import `debtValue` into the feeds reducer from the plugin. I was able to import it, and then export it from the index file of the MCD plugin, and that seems to work. Is there a better way to import it directly from the `math.js` file?

Reply:
yes, you can do:
```js
import { debtValue } from '@makerdao/dai-plugin-mcd/dist/math';
```

Reply:
Actually, I was wrong about `debtValue` working out of the box for me. I'm confused why we are  first calling `art = MDAI.wei(art)`, but returning `art.times(rate).shiftedBy(-27)` . In the example I've been working with, `MDAI.wei(art)` returns the correct value, so shifting by -27 returns a negative number (where rate is 1).